AuthenticatorBridgeKit/Future+Extensions.swift

```diff
@@ -0,0 +1,21 @@
+import Combine
+
+extension Future {
+    /// Initialize a `Future` with an async throwing closure.
+    ///
+    /// - Parameter attemptToFulfill: A closure that the publisher invokes when it emits a value or
+    ///     an error occurs.
+    ///
+    convenience init(_ attemptToFulfill: @Sendable @escaping () async throws -> Output) where Failure == Error {
```

Comment:
🤔 Can we test this directly, instead of just covering it through other places as well? Would it make sense to do that?

Reply:
Same as above. I don't see any tests currently in BitwardenShared for this, but I'm open if we think it needs direct tests.

Reply:
Same as above, we're probably good, just wanted to double-check.